packages/kotti-ui/source/kotti-value-label/types.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from 'zod'
+
+import { KottiField } from '../kotti-field/types'
+
+export namespace KottiValueLabel {
+	export const validationSchema = z.union([
+		KottiField.Validation.errorSchema,
+		KottiField.Validation.successSchema,
+		KottiField.Validation.warningSchema,
+	])
+
+	export const propsSchema = z.object({
+		dataTest: z.string().nullable().default(null),
+		helpDescription: z.string().nullable().default(null),
+		helpText: z.string().nullable().default(null),
+		isLoading: z.boolean().default(false),
+		isUnset: z.boolean().default(false),
+		label: z.string().nullable().default(null),
+		validation: validationSchema.nullable().default(null),
+		value: z.union([z.number(), z.string()]).nullable().default(null),
```

Comment:
1. why not boolean? 
2. why not an array of values as well? where the value can be comma-separated or hyphen separated? 
3. why not date? we support DisplayDate

Reply:
Current scope was to support text values only with other values (number, currency, date) as follow-up work. We need to align on this.

Reply:
Fair enough